src/main/java/Storage.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Storage class to represent the tasks being stored into hard drive.
+ * Contains file location of saved task and the file f representing the saved tasks.
+ */
+public class Storage {
+
+    private String fileLocation = "./Data/Tasks.txt"; //hard-coded relative file location of stored tasks
+    private File file;
+
+    // initialization of Storage
+
+    /**
+     * Constructor for the storage class.
+     */
+    public Storage() {
+        this.file = new File(this.fileLocation);
+    }
+
+    /**
+     * Function to check if the file exists or not.
+     * @return true if it exists, else false.
+     */
+    public boolean existFile() {
+        return this.file.exists();
+    }
+
+    /**
+     * Function to retrieve all tasks from the storage.
+     * @return ArrayList representing all the tasks from past interactions with Duke.
+     */
+    public ArrayList<Task> getTaskFromStorage() {
```

Comment:
Maybe you want to change the return type to TaskList? The main point of creating a TaskList class is to do away with using ArrayList<Task>.